mlflow/java/client/src/main/java/org/mlflow/tracking/MlflowClient.java

```diff
@@ -348,6 +348,8 @@ public ExperimentsPage searchExperiments(String searchFilter,
     SearchExperiments.Response response = mapper.toSearchExperimentsResponse(ojson);
     return new ExperimentsPage(response.getExperimentsList(), response.getNextPageToken(),
       searchFilter, experimentViewType, maxResults, orderBy, this);
+<<<<<<< HEAD
+=======
```

Comment:
Can we fix this?

Reply:
working on a huge list of fixes right now :)